scene/resources/primitive_meshes.h

```diff
@@ -106,17 +106,23 @@ class PrimitiveMesh : public Mesh {
 class CapsuleMesh : public PrimitiveMesh {
 	GDCLASS(CapsuleMesh, PrimitiveMesh);
 
+private:
+	static constexpr int default_radial_segments = 64;
+	static constexpr int default_rings = 8;
+
 private:
```

Comment:
This `private:` is now redundant with the one added above (same for other classes).

I'm not 100% convinced that adding the constants is needed here to avoid repeating them in both the member initialization and the `create_mesh_array` signature. I think they're close enough that it's understandable that a hardcoded `64` in both is meant to be kept in sync. But I'm not strongly opposed to it either.

Reply:
done